### DISCUSSION TITLE
WithCronSchedule() does not parse the cron expression correctly

### DISCUSSION THREAD

**Discussion by IT-VBFK** [quartznet/quartznet#1478] [Q&A] [answered]
**Describe the bug**

If I run following configuration in AspNetCore

```csharp
services.AddQuartz(q => {
    q.ScheduleJob<TestJob>(trigger => trigger
        .WithIdentity("Test Trigger")
        .StartNow()
        .WithCronSchedule("*/10 * * * * ?")
    );
});
```
this results in a job which would run every 10 seconds. (instead of every 10th minute).

What I am doing wrong here?

**Version used**

3.3.3

**To Reproduce**

```csharp
public class TestJob : IJob {
    public async Task Execute(IJobExecutionContext context) {
        await Console.Out.WriteLineAsync(DateTime.Now.ToString());
    }
}

```

Startup.cs
```csharp
services.AddQuartz(q => {
    q.ScheduleJob<TestJob>(trigger => trigger
        .WithIdentity("Test Trigger")
        .StartNow()
        .WithCronSchedule("*/10 * * * * ?")
    );
});
services.AddQuartzServer(options=> {
    options.WaitForJobsToComplete = true;
});
```

Console Output:

<img width="187" alt="image" src="https://user-images.githubusercontent.com/49762557/147347412-db963589-f83d-438b-a599-772a248d7614.png">

**Expected behavior**

Job is running every 10th minute

<img width="970" alt="image" src="https://user-images.githubusercontent.com/49762557/147347477-67738d6f-e99d-4147-b6b9-9d4640e7d14c.png">

**Additional context**

**Comment by lahma** [comment, marked as answer]
First field is seconds, use http://www.cronmaker.com/ to build Quartz-like expressions.

**Reply by IT-VBFK** [reply]
Ahh i see
Thank you for your help 👍🏻
